EDDI/MainWindow.xaml.cs

```diff
@@ -157,6 +157,9 @@ int centerWindow(int measure, int defaultValue)
 
         public MainWindow()
         {
+            SplashScreen splashScreen = new SplashScreen("logo.png");
+            splashScreen.Show(true);
```

Comment:
Please wrap the SplashScreen code like this to prevent a crash when running as a VoiceAttack plugin:
```cs
            if (!App.FromVA)
            {
                SplashScreen splashScreen = new SplashScreen("logo.png");
                splashScreen.Show(true);
            }
```